### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ajaysngh040**
need to merge with main branch

### CODE REVIEW

**Review by ajayjaswal76 — APPROVED** [review]
everything is fine

**Review by ajayjaswal76 — APPROVED** [review]
approved

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
-# edgeledger-web-project
-Financial website which i created while i was learning HTML and CSS basics.  Utilized all my knowledge to complete this project .  Helped me with basic html structure that we use for different sections and different CSS properties like flexbox.
```
Comment (ajayjaswal76) [inline README.md:2]: readme file

**Review by ajayjaswal76 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ajayjaswal76) [reply README.md:2]: noted

Comment (ajaysngh040) [reply README.md:2]: ok

**Review by ajayjaswal76 — APPROVED** [review]
done

**Pull request merged**